src/main/java/Handler/FileHandler.java

```diff
@@ -0,0 +1,73 @@
+import java.io.BufferedReader; 
+import java.io.File; 
+import java.io.FileReader; 
+import java.io.IOException; 
+ 
+public class FileHandler implements Handler { 
+  String directoryPath; 
+  String filePath; 
+  Request request; 
+ 
+  public FileHandler(String directoryPath) throws NonexistentDirectoryException { 
+    File directory = new File(directoryPath);
```

Comment:
This works fine and I think it's okay for now, but now your file handler is coupled to using a system directory as a static resource store. What if we want to handle files that are stored elsewhere, like in a database? How would you decouple the FileHandler from using these specific methods to read files from the system?